lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.h

```diff
@@ -108,6 +108,9 @@ class DWARFASTParserClang : public lldb_private::plugin::dwarf::DWARFASTParser {
   lldb_private::ConstString GetDIEClassTemplateParams(
       const lldb_private::plugin::dwarf::DWARFDIE &die) override;
```

Comment:
delete `lldb_private::plugin::dwarf::` (and elsewhere in this file)

Reply:
Done.